nox/__main__.py

```diff
@@ -96,6 +98,9 @@ def merge_from_options(self, options):
             options.error_on_external_run,
             self.no_error_on_external_run,
         )
+        self.install_only = _default_with_off_flag(
+            self.install_only, options.install_only, self.no_install_only
```

Comment:
I don't forsee users wanting to set `install only` in their Noxfiles, so it's probably fine to make this a non-noxfile-changable-flag. We can always add it alter.

Reply:
Done.